pycaret/__init__.py

```diff
@@ -1,7 +1,26 @@
+import sys
+
 from pycaret.utils._show_versions import show_versions
 
 version_ = "3.1.0"
 
 __version__ = version_
 
 __all__ = ["show_versions", "__version__"]
+
+# Pycaret only supports python 3.8, 3.9, 3.10
+# This code is to avoid issues with python 3.7 or other not supported versions
+# example (see package versions): https://github.com/pycaret/pycaret/issues/3746
+
+if sys.version_info < (3, 8):
+    raise Exception(
+        "Pycaret only supports python 3.8, 3.9, 3.10. Your actual Python version: ",
+        {sys.version_info},
```

Comment:
Why the brackets? This shows the brackets in the exception, like this `Python version: {3.8}`, which doesn't look as clean as without imo. You can either remove them or use f-strings.

Reply:
ok